src/components/Overlay/OfficeOverlay.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+    <div :class="classNames.root">
+        <slot/>
+    </div>
+</template>
+
+<script lang="ts">
+    import {Component, Vue, Prop} from "vue-property-decorator";
+    import OfficeLabel from "@/components/Label/OfficeLabel.vue";
+    import OfficeIcon from "@/components/Icon/OfficeIcon.vue";
+    import {IStyleFunctionOrObject, mergeStyles, mergeStyleSets} from "@uifabric/merge-styles";
+    import {getStyles} from "@/components/Overlay/OfficeOverlay.style";
+    import {createTheme} from "@/styling";
+    import {IOfficeOverlayStyleProps, IOfficeOverlayStyles} from "@/components/Overlay/OfficeOverlay.types";
+    import {getDocument} from "@/utility/dom";
+
+
+    @Component({
+        components: {OfficeLabel, OfficeIcon}
+    })
+    export default class OfficeOverlay extends Vue {
+        @Prop({type: Object}) private styles?: IStyleFunctionOrObject<IOfficeOverlayStyleProps, IOfficeOverlayStyles>;
+        @Prop({type: Boolean, default: undefined}) private className!: string;
```

Comment:
if this can be undefined, then don't use the `!` operator (`className!`), but use the `?` operator (`className?`) instead!

Reply:
Also we don't need the `className` prop at all in vue, so you can simply remove it